src/EA.Weee.Web/Views/Shared/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -29,7 +38,8 @@
 </div>
 <div class="ui-helper-clearfix"></div>
 <h2 class="govuk-heading-m govuk-!-padding-top-5 govuk-!-margin-bottom-2">Total transferred (tonnes)</h2>
-<table id="transfer-note-tonnage-summary" class="govuk-table">
+
+<table id="transfer-note-tonnage-summary" class="govuk-table" style="width:88%">
```

Comment:
should the 88% width be on this table? Not sure if this would conflict with what Nathan is doing as he uses the same view.

Reply:
I will check with Nathan first